cmd/wc/main.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+	"unicode/utf8"
+)
+
+func main() {
+	var lPtr, wPtr, cPtr bool
```

Comment:
this is not pointer, the naming should be more descriptive

Reply:
apply for all

Reply:
> apply for all

Done